js/src/ui/AccountCard/accountCard.js

```diff
@@ -28,6 +28,7 @@ import styles from './accountCard.css';
 export default class AccountCard extends Component {
   static propTypes = {
     account: PropTypes.object.isRequired,
+    allowAddressClick: PropTypes.bool,
```

Comment:
I know it's not very common practice, but could we add `static defaultProps` with this `allowAddressClick` set to `false` ? I think it would be a bit more obvious what the default behavior is.